### PR TITLE
course campaign select list ordered by create time, waffle #1142

### PR DESCRIPTION
**Pull request by patmbolger**
On the course page a course can be associated with a campaign.  This story consists of:

1) Present the list of campaigns in time-sorted order, most recent at top of list
2) Add test expectations for this behavior

### CODE REVIEW

**Comment by ragesoss** [comment]
LGTM! I'll merge it as soon as the tests pass.

File: spec/features/admin_role_spec.rb [hunk 1]
```diff
@@ -70,7 +72,14 @@
 
       # Edit details and add campaign
       click_button('Edit Details')
+
       page.all('.button.border.plus')[4].click
+
+      # Ensure campaigns appear in select list ordered by time (descending)
+      campaign_options = all('select[name=campaign]>option')[1,2]
```
Comment (sejalkhatri) [inline spec/features/admin_role_spec.rb]: can u explain this line, just for info!

Comment (ragesoss) [inline spec/features/admin_role_spec.rb]: `all('select[name=campaign]>option')` returns an array of the options in that select element.

`array[1,2]`returns a portion of the array, starting from index 1, with length 2. https://docs.ruby-lang.org/en/2.3.0/Array.html#method-i-5B-5D

Comment (sejalkhatri) [inline spec/features/admin_role_spec.rb]: Thanks @ragesoss  :+1:

**Pull request merged**